go/test/endtoend/vreplication/movetables_mirrortraffic_test.go

```diff
@@ -25,6 +25,14 @@ import (
 
 func testMoveTablesMirrorTraffic(t *testing.T, flavor workflowFlavor) {
 	setSidecarDBName("_vt")
+	ogReplicas := defaultReplicas
```

Comment:
Nit: "og" naming in this function as compared with "orig" naming in other functions. I prefer the latter FWIW.

Reply:
We are using this pattern cut-pasted in many places. This is going to be refactored soon into calling a function that changes the number of replica/rdonly count and returns a `defer`able reset function. (Or we will remove the globals and make them test-scoped)

So merging as-is for now to get the backport ready well in time for RC2.